src/applications/hca/containers/IntroductionPageGated.jsx

```diff
@@ -114,7 +116,13 @@ const LoggedOutContent = connect(
       downtime={route.formConfig.downtime}
     />
     <div className="omb-info--container" style={{ paddingLeft: '0px' }}>
-      <OMBInfo resBurden={30} ombNumber="2900-0091" expDate="05/31/2018" />
+      {environment.isProduction() ? (
```

Comment:
adding the gating logic here is fine, but this component itself is gated in the form config.

Reply:
that's fine- I don't want it on production. I'll add it to both components once we go to production if it's still necessary